src/main/kotlin/app/revanced/patches/twitter/misc/featureoverride/fingerprints/FeatureOverrideFingerprint.kt

```diff
@@ -0,0 +1,26 @@
+package app.revanced.patches.twitter.misc.featureoverride.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.music.layout.minimizedplayback.annotations.MinimizedPlaybackCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+@Name("feature-override-fingerprint")
+@MinimizedPlaybackCompatibility
+@Version("0.0.1")
+object FeatureOverrideFingerprint : MethodFingerprint(
```

Comment:
This fingerprint seems like it could easily match to another method. Consider making it stronger.